lib/web_console/middleware.rb

```diff
@@ -52,7 +52,7 @@ def call(env)
     private
 
       def acceptable_content_type?(headers)
-        headers["content-type"].to_s.include?("html")
+        headers[Rack::CONTENT_TYPE].to_s.include?("html")
```

Comment:
There's another "content-type" 5 lines below

Reply:
The other one was not causing an issue, since we had no conditional on it, but I updated it and the other references too for the sake of consistency. They were mostly tests.

I left this one: 
https://github.com/rails/web-console/blob/main/lib/web_console/templates/console.js.erb#L938
I see it as separate since It's on the request side. Just like we can't control the casing from a browser request.
It's in an ERB so we could use same constant, but it would make the code a bit uglier. 
Do you prefer to change it too?

Reply:
No, I agree with you that one should be fine since its a request header (Rack 3 only requires response headers to be downcasesd)